website/docs/patterns/disabled-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,74 @@
+### Permissions 
+
+When a user does not have CRUD (Create, Read, Update and Delete) permissions, hide the related actions.
+
+In this example, the user's role is **viewer** and they cannot **deploy** or **destroy** a cluster. An explanation is not needed as users understand that their account has limited permissions and that they require an additional user's help to gain these permissions (often someone with admin or similar roles).
+
+![Image showing two dropdown with lists open, one with only one menu item that is reader specific, while the other has more options that allow for further capabilities.](/assets/patterns/disabled-patterns/permissions-example.png)
+
+In scenarios where a user lands on a page they cannot access, provide a reason why with guidance so that users understand what steps they can take to rectify the error. So that users don't feel stuck, provide them a call to action to facilitate a path forward.
+
+![Showing a 403 error page with information about what is going on, with a way to go back and a way to log in with a different token](/assets/patterns/disabled-patterns/permissions-example-3.png)
+
+### Conditional availability
+
+When a feature or function is conditionally available depending on other steps that must be taken, provide context in place of that action so that users are able to take those steps to enable this feature.
```

Comment:
This still reads very strange to me and is a bit jumbled. I _think_ I understand what you're trying to say, let me take a stab:

```suggestion
When a feature or function is unavailable due to unmet conditions, provide context in the form of text or an [Alert](/components/alert) guiding user through enabling it.
```

Could also alter the last part to be:

"...provide context or instructions guiding the user through enabling it." dropping the specific component/element usage.